CMakeLists.txt

```diff
@@ -87,6 +87,9 @@ if(${USE_Z3_DLOPEN})
 elseif (${Z3_FOUND})
   add_definitions(-DHAVE_Z3)
   message("Z3 SMT solver found. This enables optional SMT checking with Z3.")
+  if("${Z3_VERSION}" VERSION_LESS "4.8.12.0")
+	message(WARNING "SMT tests need Z3 4.8.12 or higher, SMT tests are expected to fail!")
```

Comment:
missing indentation here

Reply:
indentation for the line 91 looks good, but I used tab I will change it to 4 spaces.
and I should add a space after `if`